python/tests/core/test_online_store_rest_client_engine.py

```diff
@@ -0,0 +1,345 @@
+# #
+# #   Copyright 2024 Hopsworks AB
+# #
+# #   Licensed under the Apache License, Version 2.0 (the "License");
+# #   you may not use this file except in compliance with the License.
+# #   You may obtain a copy of the License at
+# #
+# #       http://www.apache.org/licenses/LICENSE-2.0
+# #
+# #   Unless required by applicable law or agreed to in writing, software
+# #   distributed under the License is distributed on an "AS IS" BASIS,
+# #   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# #   See the License for the specific language governing permissions and
+# #   limitations under the License.
+# #
+# import humps
+# import pytest
+# from datetime import datetime
+
+# from hsfs.core import online_store_rest_client_engine
+
+# RONDB_REST_API_GET_BATCH_RAW_FEATURE_VECTORS = "hsfs.core.online_store_rest_client_api.OnlineStoreRestClientApi.get_batch_raw_feature_vectors"
+
+
+# class TestOnlineRestClientEngine:
```

Comment:
Why is this class completely commented?

Reply:
Because I had not fix those tests in the version that requires passing serving keys to this class. Development this week has been happening in a different branch using the updated rondb server image